src/db/tables/subscribers.js

```diff
@@ -193,6 +193,27 @@ async function setAllEmailsToPrimary (subscriber, allEmailsToPrimary) {
 }
 /* c8 ignore stop */
 
+/**
+ * @param {import("knex/types/tables").SubscriberRow} subscriber
+ * @param {boolean | null} monthlyMonitorReport
```

Comment:
Not sure I understand - why should it be possible for this value to be `null`? (And related: why do we need `|| false` on line 206? i.e. if the reason is that we want to be able to distinguish between users who have actively chosen whether to receive the email, and those who haven't made a choice, then shouldn't we also store the `null` in the DB?)

```suggestion
 * @param {boolean} monthlyMonitorReport
```